feat(ai): support text-to-speech capabilities (single & multi-speaker configurations) for non-Live API models (#10145)
firebase/firebase-js-sdk: docs-devsite/ai.basespeechconfig.md

Project: /docs/reference/js/_project.yaml
Book: /docs/reference/_book.yaml
page_type: reference

{% comment %}
DO NOT EDIT THIS FILE!
This is generated by the JS SDK team, and any local changes will be
overwritten. Changes should be made in the source code at
https://github.com/firebase/firebase-js-sdk
{% endcomment %}

# BaseSpeechConfig interface
Base configuration for speech synthesis.

<b>Signature:</b>

```typescript
export interface BaseSpeechConfig 
```

## Properties

|  Property | Type | Description |
|  --- | --- | --- |

> |  [languageCode](./ai.basespeechconfig.md#basespeechconfiglanguagecode) | string | The optional IETF BCP-47 language code (e.g., "en-US", "es-ES") to guide voice generation. |

Reviewer: ```suggestion
|  [languageCode](./ai.basespeechconfig.md#basespeechconfiglanguagecode) | string | The optional IETF BCP-47 language code (for example, `"en-US"` or `"es-ES"`) to guide voice generation. |
```

Author: Done; changed to the wording in the design doc for consistency.

## BaseSpeechConfig.languageCode

> The optional IETF BCP-47 language code (e.g., "en-US", "es-ES") to guide voice generation.

Reviewer: ```suggestion
The optional IETF BCP-47 language code (for example, `"en-US"` or `"es-ES"`) to guide voice generation.
```
